src/garage/torch/policies/deterministic_mlp_policy.py

```diff
@@ -64,7 +65,7 @@ def get_action(self, observation):
 
         """
         with torch.no_grad():
-            x = self.forward(observation.unsqueeze(0))
+            x = self.forward(torch.Tensor(observation).unsqueeze(0))
```

Comment:
shouldn't this accept numpy array, not torch.Tensor

Reply:
also, shouldn't it return ndarray?

Reply:
same for get_actions

Reply:
Done.